templates/deploy.magento2.php

```diff
@@ -14,6 +14,8 @@
 $productionStage->addServer('appname.hypernode.io');
 
 $testStage = $configuration->addStage('test', 'example.com');
-$testStage->addBrancherServer('appname');
+$testStage->addBrancherServer('appname')
+    ->setLabels(['stage=test', 'ci_ref=' . \getenv('GITHUB_RUN_ID') ?: 'none'])
```

Comment:
These labels are all formatted as `X=Y`, but there isn't some validation based on this format right? Any format of label is allowed?

Reply:
Any format is allowed indeed